libs/interpret-text/src/lib/TextExplanationDashboard/Control/BarChart.tsx

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { localization } from "@responsible-ai/localization";
+import { AccessibleChart, IPlotlyProperty } from "@responsible-ai/mlchartlib";
+import React from "react";
+
+import { Utils } from "../CommonUtils";
+import { IChartProps } from "../Interfaces/IChartProps";
+
+export class BarChart extends React.PureComponent<IChartProps> {
+  /*
+   * returns an accessible bar chart from mlchartlib
+   */
+  public render(): React.ReactNode {
+    return (
+      <AccessibleChart
```

Comment:
let's check with bo on high chart, if he prefer we can start using high chart here.

Reply:
done, updated to use highcharts now!